src/libraries/Native/Unix/System.Native/pal_time.h

```diff
@@ -43,6 +43,8 @@ DLLEXPORT uint64_t SystemNative_GetTimestamp(void);
  * for the CLR thread pool to regulate the number of worker threads.
  * Since there is no consistent API on Unix to get the CPU utilization
  * from a user process, getrusage and gettimeofday are used to
- * compute the current process's CPU utilization instead.
+ * compute the current process's CPU utilization instead. The CPU utilization
+ * returned is sum of utilization accross all processors, e.g. this function will
+ * return 200 when two cores are running at 100%.
```

Comment:
I'm a little confused by removing the multiplication by number of processors, but then this saying it's larger for more processors. I'm sure you're right, but what's my misunderstanding?

Reply:
System.Native shim does not have easy access to the process count adjusted for container limits. The processor count on managed size is adjusted and thus it is the right number to use to compute how much spare cycles may extra threads take advantage of.

In other words, this change should get us equivalent of [this]( https://github.com/dotnet/runtime/blob/master/src/coreclr/src/pal/src/thread/process.cpp#L2552) piece of logic from CoreCLR PAL.

Reply:
It's essentially removing division, not multiplication. It's not obvious from reading the unified diff because GitHub hides the `cpuUtilization = (int32_t)(cpuBusyTime * 100 / cpuTotalTime);` line. The multiplication in `cpuTotalTime` is affecting the divisor.

Reply:
Ah, yeah, I missed that, thanks.